exorelay-javascript/package.json

```diff
@@ -4,27 +4,27 @@
   "author": "Kevin Goslar",
   "dependencies": {
     "chalk": "1.1.3",
-    "debug": "2.6.0",
+    "debug": "2.6.6",
     "rails-delegate": "0.6.2",
     "uuid": "3.0.1",
     "wait": "0.1.0",
-    "ws": "2.0.3"
+    "ws": "2.3.1"
   },
   "description": "Communication relay between JavaScript code bases and the Exosphere environment",
   "devDependencies": {
     "chai": "3.5.0",
     "cucumber": "1.3.1",
     "cucumber-snippets-livescript": "1.0.1",
     "dependency-lint": "4.3.1",
-    "ejs": "2.5.5",
+    "ejs": "2.5.6",
     "exocom-mock": "0.21.4",
```

Comment:
Unrelated to this PR, but I think this makes versioning a bit tricky. If we make changes in both `exorelay` and `exocom-mock`, those changes will get published in `0.21.5`, but `exorelay@0.21.5` will have been testing against an older version of `exocom-mock@0.21.4`. 

This is a similar issue we had with `exo-relay` and `exo-service` both needing to be lock-versioned but also depending on each other's latest changes. Although maybe it's not as big of an issue since `exocom-mock` is only used for tests in development.